dolphin/api/v1/router.py

```diff
@@ -49,6 +50,21 @@ def _setup_routes(self, mapper):
                        controller=self.resources['access_info'],
                        action="update",
                        conditions={"method": ["PUT"]})
+
+        self.resources['alert_sources'] = alert.create_resource()
+        mapper.connect("storages", "/storages/{id}/alert_sources",
```

Comment:
plz don't use underscore in the url.  suggest name: alert-source

Reply:
Fixed